server/app/views/NorthStarBaseView.java

```diff
@@ -82,6 +82,24 @@ protected ThymeleafModule.PlayThymeleafContext createThymeleafContext(
     context.setVariable("closeIcon", Icons.CLOSE);
     context.setVariable("httpsIcon", assetsFinder.path("Images/uswds/icon-https.svg"));
     context.setVariable("govIcon", assetsFinder.path("Images/uswds/icon-dot-gov.svg"));
+    context.setVariable("baseUrlWithoutScheme", getBaseUrlWithoutScheme());
+    String CommonIntakeMoreResourceLink =
+        settingsManifest.getCommonIntakeMoreResourcesLinkHref(request).get();
+    // In Thymeleaf, it's impossible to add escaped text inside unescaped text, which makes it
+    // difficult to add HTML within a message. So we have to manually build the html for a link
+    // that will be embedded in the more resources section of the footer.
+    if (CommonIntakeMoreResourceLink != null && !CommonIntakeMoreResourceLink.isEmpty()) {
+      context.setVariable(
+          "moreResourcesLink",
+          "<a id=\"more-resources-link\" class=\"text-white underline font-bold\" href=\""
+              + CommonIntakeMoreResourceLink
+              + "\">"
+              + "Visit "
+              + CommonIntakeMoreResourceLink
+              + "</a>");
+    } else {
+      context.setVariable("moreResourcesLink", null);
+    }
```

Comment:
(blocking) We need to go with a solution that isn't concatenating string to build the html. Based on what I see in the issue you should be able to move all the formatting in to the thymeleaf directly. 

The `Looking for more informationservices?` should be formatted separately from the `Visit cityorcounty` part. That 'Visit' part will be able to completely be a link. This will remove the need for the string concatentation as well as the need for using `utext`.